src/system/Wizard/WizardStep.js

```diff
@@ -67,6 +81,7 @@ const WizardStep = React.forwardRef(
 							fontSize: 2,
 							fontWeight: active ? 'bold' : 'normal',
 						} }
+						ref={ titleRef }
```

Comment:
I think you forgot to add this to make the element focusable.
`tabIndex={ -1 }`

Reply:
@djalmaaraujo @kat3samsin added the optional TabIndex in https://github.com/Automattic/vip-design-system/pull/175/commits/e39c3946c2a452ac7213d510fae1217be3855d70

Thanks!